core-bundle/tests/Command/InstallCommandTest.php

```diff
@@ -97,4 +109,42 @@ public function testHandlesCustomFilesAndImagesPaths()
         $this->assertContains(' * files_test', $display);
         $this->assertContains(' * assets/images_test', $display);
     }
+
+    /**
+     * Tests adding the initialize.php file and tcpdf.php symlink.
+     */
+    public function testCreatesInitializeFileAndTcpdfSymlink()
+    {
+        $command = new InstallCommand(
+            $this->getRootDir(),
+            'files_test',
+            $this->getRootDir().'/assets/images_test',
+            $this->getBundlesMetadata()
+        );
+
+        $tester = new CommandTester($command);
+        $tester->execute([]);
+
+        $this->assertFileExists($this->getRootDir().'/system/initialize.php');
+
+        /*
+         * We need to check both if the tcpdf.php is a symlink and is in fact a file,
+         * because is_link() returns true, even if the symlink is invalid and
+         * file_exists() returns false, if the symlink is invalid.
+         */
```

Comment:
```suggestion
        // is_link() returns true even if the symlink target does not exist, therefore also check file_exists()
```

Would be shorter. Either way is fine though.